packages/odie-client/src/data/use-send-odie-message.ts

```diff
@@ -205,13 +225,30 @@ export const useSendOdieMessage = ( signal: AbortSignal ) => {
 				return;
 			}
 
-			if ( ! odieId ) {
-				addEventToInteraction.mutate( {
-					interactionId: currentSupportInteraction!.uuid,
-					eventData: {
-						event_external_id: returnedChat.chat_id.toString(),
+			const chatId = returnedChat.chat_id;
+			let supportInteraction = currentSupportInteraction;
+
+			try {
+				if ( ! supportInteraction && chatId ) {
+					supportInteraction = await startNewInteraction( {
+						event_external_id: chatId.toString(),
 						event_source: 'odie',
-					},
+					} );
+				} else if ( supportInteraction && ! odieId && chatId ) {
+					supportInteraction = await addEventToInteraction.mutateAsync( {
+						interactionId: supportInteraction.uuid,
+						eventData: {
+							event_external_id: chatId.toString(),
+							event_source: 'odie',
+						},
+					} );
+				}
+			} catch ( error ) {
+				trackEvent( 'error_updating_support_interaction', {
+					error_message:
+						error instanceof Error ? error.message : error?.toString?.() ?? 'Unknown error',
+					has_existing_interaction: Boolean( supportInteraction?.uuid ),
+					has_chat_id: Boolean( chatId ),
```

Comment:
Probably more useful to include the ids
```suggestion
					existing_interaction: supportInteraction?.uuid ?? '',
					chat_id:  chatIdd ?? '',
```